test/supp/helgrind-umf_test-ipc.supp

```diff
@@ -14,3 +14,40 @@
    fun:critnib_find
    ...
 }
+
+{
+   [false-positive] Double check locking pattern in trackingOpenIpcHandle
```

Comment:
Do you add extra false positives in this patch? This looks supspicius

Reply:
We already have the same suppression for IPC tests with other providers (for example see `drd-umf_test-provider_file_memory_ipc.supp`). Looks like the same false positive started to reproduce with other providers on the same IPC tests.